### DISCUSSION TITLE
내보내기 요청에 -810이 리턴됩니다

### DISCUSSION THREAD

**Discussion by kiwioverflow** [storycraft/node-kakao#490] [Q&A]
한번에 여러명을 내보내려고 하면 `{"status":-810,"result":false}` 이렇게 리턴됩니다.

이후 내보내기 요청을 보내도 저거만 리턴되네요

봇 재시작을 해야만 해결 되는 듯 한데...

혹시 관련해서 뜻이나 해결방법 아시는 분 없으실까요?

**Comment by generalxd** [comment]
한번에 여러명을 내보내기 하면안됩니다..

**Comment by kiwioverflow** [comment]
더불어서 한번 내보내진 사람을 다시 내보내려고 해도 -810이 리턴되네요
